### PR TITLE
Add bandit API document

### PR DESCRIPTION
**Pull request by TkrUdagawa**
This pull request is related to #183 
I have added Japanese and English document of Bandit algorithm API.

### CODE REVIEW

File: source/en/api_bandit.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      :return:     ``arm_id`` selected by bandit algorithm.
+
+      Select player's arm according to current state.
+
+   .. mpidl:method:: bool register_reward(0: string player_id, 1: string arm_id, 2: double reward)
+
+      :param player_id: ID of the player whose arm gets rewards
+      :param arm_id:    ID of the arm which rewards are registered with
+      :param reward: 	amount of rewards
+      :return:          True if succeeded in registring reward. False if failed to register rewards.
```
Comment (kumagi) [inline source/en/api_bandit.rst]: s/registring/registering/

File: source/en/api_bandit.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+   .. mpidl:method:: map<string, arm_info> get_arm_info(0: string player_id)
+
+      :param target_label: ID of the player
+      :return:          arm information of specified player
+
+      Get all arms information of specified player.
+
+   .. mpidl:method:: bool reset(0: string player_id)
+      :param target_label: ID of the user whose arms are to be reset.
+      :return:          　 True if succeeded in reseting the arm. False if failed to reset.
```
Comment (kumagi) [inline source/en/api_bandit.rst]: s/reseting/resetting/

File: source/en/api_bandit.rst [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Configuration
+~~~~~~~~~~~~~
+
+Configuration is given as a JSON file.
+We show each field below:
+
+.. describe:: method
+
+   Specify bandit algorithm.
+   You can use algorithms below.
```
Comment (kumagi) [inline source/en/api_bandit.rst]: may be good "You can use below algorithms."

Comment (TkrUdagawa) [inline source/en/api_bandit.rst]: I think "the algorithms below" is better.
A good example of "below" is the sentence from English tests such as TOEIC: "Read the following dialogue and answer the questions below".

File: source/en/api_bandit.rst [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+      ==================== ===================================
+
+.. describe:: parameter
+
+   Specify parameters for the algorithm.
+   Its format differs for each ``method``.
+
+   epsilon_greedy
+     :epsilon:
+         The probability of choosing arms randomly.
```
Comment (kmaehashi) [inline source/en/api_bandit.rst]: Strangely indented.

File: source/en/api_bandit.rst [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+     :gamma:
+	Mixture rate of constant weight and each arm's weight.
+	The higher ``gamma`` is, the higher the rate of constant weight is.
+	The lower ``gamma`` is, the higher the rate of each arm's weight is.
+	(Float)
+
+        * Range: 0.0 <= ``gamma`` <= 1.0
+
+
+例:
```
Comment (kmaehashi) [inline source/en/api_bandit.rst]: ＿人人人人人人＿
＞　突然の例　＜
￣Y^Y^Y^Y^Y^￣

**Comment by kmaehashi** [comment]
Commit logs are dirty.  Cloud you please rebase to the latest develop branch?

**Comment by kmaehashi** [comment]
@gintenlabo document assume_unrewarded please

**Comment by gintenlabo** [comment]
I will write document for `assume_unrewarded` option in another pulll request.

**Comment by kmaehashi** [comment]
> I will write document for assume_unrewarded option in another pulll request.

@gintenlabo Please raise a new issue for that to avoid forgetting it.

**Pull request merged**